src/datadog/datadog_agent.cpp

```diff
@@ -215,15 +215,24 @@ void DatadogAgent::flush() {
                       logger = logger_](int response_status,
                                         const DictReader& /*response_headers*/,
                                         std::string response_body) {
-    if (response_status < 200 || response_status >= 300) {
+    if (response_status != 200) {
       logger->log_error([&](auto& stream) {
-        stream << "Unexpected response status " << response_status
-               << " with body (starts on next line):\n"
+        stream << "Unexpected response status from Datadog Agent"
```

Comment:
Let's put a space after the word "Agent" so that the response status is surrounded by spaces.

Reply:
Come to think of it, consider the following order:
```c++
stream << "Unexpected response status " << response_status
       << " in Datadog Agent response with body of length "
       << response_body.size() << " (starts on next line):\n"
       << response_body;
```
Might as well say it all.

Reply:
Good catch. Fixed in 8c405b49c68a5c20b5417a1c76261e1e49672b75